internal/pkg/stream/ecs.go

```diff
@@ -23,7 +23,7 @@ const (
 	rollOutEmpty               = ""
 )
 
-var ecsEventFailureKeywords = []string{"fail", "unhealthy", "error", "throttle", "unable", "missing"}
+var ecsEventFailureKeywords = []string{"fail", "unhealthy", "error", "throttle", "unable", "missing", "alarm", "rolling"}
```

Comment:
what do you think of `"alarm detected"` and `"rolling back"`? Just so that we don't accidentally catch something that happens to contain "alarm" / "rolling" (what if a service name is "rolling stone" 😂 ) idk sounds far fetch. I am fine either way 😂